packages/core/src/node-utils/files.ts

```diff
@@ -167,9 +168,11 @@ function flushDirectories(directory: string, recursive = true): void {
 }
 
 async function readFileAndCreateHash(filePath: string): Promise<string> {
-  return fs.promises
-    .readFile(filePath, "utf-8")
-    .then((file) => createHash("md5").update(file).digest("hex"));
+  return fs.promises.readFile(filePath, "utf-8").then((file) => hash(file));
+}
+
+function readFileAndCreateHashSync(filePath: string): string {
+  return hash(fs.readFileSync(filePath, "utf-8"));
```

Comment:
we check that the file exists before calling the `readFileAndCreateHashSync` but something might change in the filesystem. I'll try/catch the call just in case